src/js/components/receiving/modals/EditLineModal.jsx

```diff
@@ -108,11 +120,25 @@ class EditLineModal extends Component {
     this.setState({ attr });
   }
 
+  groupShipmentItems() {
+    const { containers } = this.props.values;
+    /**
+     * containers are built like: [{..., shipmentItems: []}]
+     * so in the end we have to flat the result,
+     * as after mapping, the result would look like
+     * [[<shipmentItem>, <shipmentItem>], [<shipmentItem>, <shipmentItem>]]
+     */
+    const shipmentItems = containers.map((container) => container.shipmentItems).flat();
```

Comment:
what about `_.flatten`?

Reply:
I don't see any advantage of that, since `flat()` does the same

Reply:
you're right, so I think you can use `_.flatMap` instead :smile:

Reply:
:thinking: I swear I checked whether the `.flatMap` exists in JS, but couldn't find it, and suddenly it exists :laughing: